antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/mode/common/binding_const.py

```diff
@@ -8,21 +8,13 @@
 
 class OutputSimulationBindingConstraintItem(FolderNode):
     def build(self) -> TREE:
-        # filters = self.config.get_filters_synthesis(self.area, self.link)
-        # todo get the config related to this output (now this may fail if input has changed since the launch)
-
-        freq: MatrixFrequency
+        existing_files = [d.stem.replace("binding-constraints-", "") for d in self.config.path.iterdir()]
         children: TREE = {
             f"binding-constraints-{freq}": BindingConstraintOutputSeriesMatrix(
                 self.context,
                 self.config.next_file(f"binding-constraints-{freq}.txt"),
-                freq,
+                MatrixFrequency(freq),
```

Comment:
J'ai un doute : les matrices pour les contraintes couplantes tiennent compte des années bissextiles. C'est-à-dire, 8784 heures au lieu de 8760, 366 jours au lieu de 365 ou 53 semaines au lieu de 52. Est-ce que cela peut avoir une influence sur la taille des matrices de sortie ?

Reply:
This PR won't change anything about this subject but it's a tricky question. We'll have to disucss it in another PR (for example when we'll show the matrices in the Results view)